src/criteria_assessment/criteria.jl

```diff
@@ -111,6 +111,19 @@ function _write_cog(file_path::String, data::Raster, config::Dict)::Nothing
     return nothing
 end
 
+function _write_tiff(file_path::String, data::Raster, config::Dict)::Nothing
```

Comment:
`config` not used?

Reply:
Ah, I think because we went from writing out a COG (which used the defined options) to writing out a geotiff...

I'll fix it up, thanks for catching.